xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESound.h

```diff
@@ -59,6 +61,7 @@ class CActiveAESound : public IAESound
   static int64_t Seek(void *h, int64_t pos, int whence);
 
 protected:
+  CActiveAE *m_activeAE;
```

Comment:
For the record. There is hardly any other component in Kodi that is self contained like AE. Even  AE related settings are owned by AE and are not accessed from the outside. In this iteration with killing the old factory it gets another improvement.

Calling AE broken in redard to the rest of the app feels somehow rediculous. In regards to dependencies the code I have maintained has improved a lot. Coming to my place and trying to reject code I wrote for code I wrote feels offending.